db/migrate/20210222102503_create_users_settings.rb

```diff
@@ -0,0 +1,25 @@
+class CreateUsersSettings < ActiveRecord::Migration[6.0]
+  def change
+    create_table :users_settings do |t|
+      t.integer :user_id, null: false
+      t.string "config_font", default: "default"
+      t.string "config_navbar", default: "default", null: false
+      t.string "config_theme", default: "default"
+      t.boolean "display_announcements", default: true
+      t.boolean "display_sponsors", default: true
+      t.string "editor_version", default: "v1"
+      t.integer "experience_level"
+      t.boolean "feed_mark_canonical", default: false
+      t.boolean "feed_referential_link", default: true, null: false
+      t.string "feed_url"
+      t.string "inbox_guidelines"
+      t.string "inbox_type", default: "private"
+      t.boolean "permit_adjacent_sponsors", default: true
+      t.string "brand_color1", default: "#000000"
+      t.string "brand_color2", default: "#ffffff"
+      t.boolean "display_email_on_profile", default: false
```

Comment:
A generic comment: a new table seems like the perfect occasion to revisit default values and nullabilities.

For example, `config_font` is never nullable in the UI, so adding `null: false` seems like a good idea (we have no concept of "no font" in the app). Same for `config_theme`, there's no "no theme option" as the default UI is a theme.

`boolean` fields also tend to be all `null: false`, again the question is: "does a null state make sense for this field?". 

I think it's worth "improving" the schema here now that we start from scratch.

Reply:
Thanks @rhymes. Can you have another look, I added nulls where I thought made sense, let me know if you see that are unnecessary or you think need more details added.